### ISSUE TITLE
FileUriExposedException on Android 7+

### ISSUE THREAD

**Issue by matheusgrieger** [apache/cordova-plugin-file#270]
So we have recently seen that Google will block updates from apps with `targetSdk` < 26. Our apps currently use SDK 23, which works fine. Since then, we were trying to update to Target SDK 26 and comply with Google's definition.

However this plugin does not work with SDK 24+, since it requires you to use `content://` addresses instead of `file:///`. I have tried to find some replacement and updated the plugin to version 6.0.1, but to no avail.

We are also using `@ionic-native/file`, which basically just wraps `cordova-plugin-file` and adds type definitions.

Whenever we try to copy a file to `cordova.file.externalApplicationStorageDirectory` the app crashes, closes, due to a `FileUriExposedException` from `ClipData.Item.getUri()`.

Is there any solution to this?

**Comment by fontpiece** [comment]
do you fixed it ?

**Comment by matheusgrieger** [comment]
@fontpiece no, we haven't. However, we removed Crosswalk from our application and it seems to be working fine by now.
